### ISSUE TITLE
cloudstack-sysvmadm - /etc/rc.d/init.d/functions: No such file or directory

### ISSUE THREAD

**Issue by dR3b** [apache/cloudstack#9000]
##### ISSUE TYPE

 * Bug Report

##### COMPONENT NAME
~~~
cloudstack-sysvmadm
~~~

##### CLOUDSTACK VERSION
~~~
CloudStack 4.19.0.1
~~~

##### OS / ENVIRONMENT
```
Ubuntu 22.04
```

##### STEPS TO REPRODUCE
Start the command: `/usr/bin/cloudstack-sysvmadm`
~~~
root@nuc1:~# cloudstack-sysvmadm
/usr/bin/cloudstack-sysvmadm: line 21: /etc/rc.d/init.d/functions: No such file or directory

  The tool for stopping/starting running system vms and domain routers

  Usage: cloudstack-sysvmadm: [-d] [-u] [-p] [-m] [-s] [-r] [-a] [-n] [-t] [-l] [-z] [-v] [-i] [-j]

   -d - cloud DB server ip address, defaulted to localhost if not specified
   -u - user name to access cloud DB, defaulted to "root" if not specified
   -p - cloud DB user password, defaulted to no password if not specified

~~~

##### EXPECTED RESULTS
~~~
No error
~~~

**Comment by DaanHoogland** [comment]
@dR3b sounds like a dependency is missing, or is this file phased out in a maintenance release of ubuntu?

**Comment by DaanHoogland** [comment]
@dR3b this is indeed an issue, but the script, cloudstack-sysvmadm, is not maintained which makes this a documentation issue. If you care you can fix the code or the docs and create a PR for it.

**Comment by dR3b** [comment]
@DaanHoogland How can I control the system-vms (router proxy)?

**Comment by dR3b** [comment]
I had a error with a script on my virtual router:
```
dns_check.py Missing entries for VMs in /etc/hosts
```
(In the meantime, I have already reinstalled the Management Server several times)

I will look into the sources 👍🏼

**Comment by DaanHoogland** [comment]
> I had a error with a script on my virtual router:
> 
> ```
> dns_check.py Missing entries for VMs in /etc/hosts
> ```

this is a typical casde for restart network with cleanup (https://cloudstack.apache.org/api/apidocs-4.19/apis/restartNetwork.html)

**Comment by DaanHoogland** [comment]
np, tell us how you fare.

**Comment by DaanHoogland** [comment]
@dR3b do you care to test the revised script from #9200 please?

**Comment by DaanHoogland** [comment]
fixed in #9200

**Comment by dR3b** [comment]
@DaanHoogland Looks good! Thanks
